templates/macros/public_draft.html

```diff
@@ -0,0 +1,16 @@
+{% macro warning(current=true, section) %}
+    {% if section and section.extra.public_draft %}
+        {% set public_draft = section.extra.public_draft %}
+        {% set issue = public_draft / 1 %}
+        {% if current %}
+            <h1 class="warning">Looks like this page is still under development</h1>
```

Comment:
I think the title here should be crisper. Something like "Draft Page". Too much noise for an `h1`. The description provides enough detail on its own.